src/components/books/addNewBook/AddNewBook.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { createBook } from '../../../redux/books/books';
+
+const AddNewBook = () => {
+  const dispatch = useDispatch();
+  const [bookInfo, setBookInfo] = useState({
+    item_id: '',
+    title: '',
+    category: '',
+  });
+  const handleChange = (name) => (e) => {
+    setBookInfo({ ...bookInfo, [name]: e.target.value });
+  };
+  const onClick = () => {
+    setBookInfo({ ...bookInfo, item_id: uuidv4() });
+  };
+  const onSubmit = (e) => {
+    e.preventDefault();
+    dispatch(createBook(bookInfo));
+    setBookInfo({ ...bookInfo, title: '', category: '' });
```

Comment:
- [ ] Like I mentioned the return is not showing the correct value, it's empty for the `title` but the categories are a mess. I see all of them  I should see only the selected one for my book.

Reply:
It fixed now